Userland/Libraries/LibJS/Tests/builtins/SharedArrayBuffer/SharedArrayBuffer.prototype.byteLength.js

```diff
@@ -0,0 +1,6 @@
+test("basic functionality", () => {
```

Comment:
Can we have a test for slice() please?

Reply:
I've copy pasted over the tests from ArrayBuffer. Doesn't feel that great. Let me know if you think I should extract these into a common file (not sure what I would call it) for these tests that can be shared between ArrayBuffer and SharedArrayBuffer